Require discoverable credentials and user verification in README examples (#501)
cedarcode/webauthn-ruby: README.md

options = WebAuthn::Credential.options_for_create(
  user: { id: user.webauthn_id, name: user.name },

>   exclude: user.credentials.map { |c| c.webauthn_id },
>   authenticator_selection: {
>     resident_key: "required",
>     user_verification: "required"
>   }

Reviewer: ```suggestion
  exclude: user.credentials.map { |c| c.webauthn_id },
  authenticator_selection: {
    resident_key: "discouraged",  # For a passwordless login or 2FA. Use "required" for a passkey-based (passwordless and usernameless) login.
    user_verification: "required" # For a passwordless or passkey-based (passwordless and usernameless) login. Use "discouraged" for 2FA.
  }
```

webauthn_credential = WebAuthn::Credential.from_create(params[:publicKeyCredential])

begin

>   webauthn_credential.verify(session[:creation_challenge], user_verification: true)

Reviewer: ```suggestion
  # Enforce user verification (pairs with the "required" request above). Omit for 2FA.
  webauthn_credential.verify(session[:creation_challenge], user_verification: true)
```

#### Initiation phase

> 
> ```ruby
> options = WebAuthn::Credential.options_for_get(
>   allow: user.credentials.map { |c| c.webauthn_id },
>   user_verification: "required"
> )

Reviewer: ```suggestion
options = WebAuthn::Credential.options_for_get(
  # Pass `allow` when the user is already known (passwordless/2FA/reauth).
  # For a passkey-based (usernameless and passwordless) login, omit it and resolve the user from `user_handle` after `from_get`.
  allow: user.credentials.map { |c| c.webauthn_id },
  user_verification: "required" # For a passwordless or passkey-based (passwordless and usernameless) login. Use "discouraged" for 2FA.
)
```

  webauthn_credential.verify(
    session[:authentication_challenge],
    public_key: stored_credential.public_key,
    sign_count: stored_credential.sign_count,

>     user_verification: true

Reviewer: ```suggestion
    sign_count: stored_credential.sign_count,
    user_verification: true # For a passwordless or passkey-based (passwordless and usernameless) login. Omit for 2FA.
```

creation_options = WebAuthn::Credential.options_for_create(
  user: { id: user.webauthn_id, name: user.name },
  exclude: user.credentials.map { |c| c.webauthn_id },

>   extensions: { appidExclude: domain.to_s },
>   authenticator_selection: {
>     resident_key: "required",
>     user_verification: "required"
>   }

Reviewer: ```suggestion
  extensions: { appidExclude: domain.to_s },
  authenticator_selection: {
    resident_key: "discouraged",  # For a passwordless login or 2FA. Use "required" for a passkey-based (passwordless and usernameless) login.
    user_verification: "required" # For a passwordless or passkey-based (passwordless and usernameless) login. Use "discouraged" for 2FA.
  }
```
